hack/build-base-images.sh

```diff
@@ -12,7 +12,7 @@ if [[ -z "${oc}" ]]; then
 fi
 
 function build() {
-  eval "'${oc}' ex dockerbuild $2 $1 ${OS_BUILD_IMAGE_ARGS:-}"
+  eval "docker build ${OS_BUILD_IMAGE_ARGS:-} -t $1 $2"
```

Comment:
Not sure, but I would rather not change more than I need to to get us unblocked.